x-pack/plugins/rollup/public/crud_app/sections/job_list/job_table/job_table.test.js

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Pager } from '@elastic/eui';
+
+import { registerTestBed } from '../../../../../__jest__/utils';
```

Comment:
Maybe we can move this `__jest__` directory into the `public` directory, since it seems unlikely that we'll be writing server-side React in the near future? Then it will be closer to the code which consumes it, and we also won't need to go up so many parent directories.

Reply:
As I quickly explained over slack, this seems to be a pattern in other plugins to have the `__jest__` folder at the root level. See for example `license_management` that also has a `util` file. I think we should stick to that convention for consistency, what do you think?